Tests/Fixtures/ClassWithTypeHintedAliasConstructorArgument.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Matthias\SymfonyServiceDefinitionValidator\Tests\Fixtures;
+
+class_alias('Matthias\SymfonyServiceDefinitionValidator\Tests\Fixtures\ExpectedClass', 'AliasedExpectedClass');
```

Comment:
Imo this statement should be part of the test. A file with a class in it, should only contain that class, no code executed while loading the file.